install/kubernetes/cilium/charts/config/templates/configmap.yaml

```
@@ -134,6 +134,12 @@ data:
   # Only effective when monitor aggregation is set to "medium" or higher.
   monitor-aggregation-flags: {{ .Values.global.bpf.monitorFlags }}
 
+{{- if .Values.global.cni.customConf }}
+
+  # Disable blacklisting of local routes not owned by Cilium.
+  blacklist-conflicting-routes: "false"
```

This should take the option value rather than `false`, ie:
```suggestion
  blacklist-conflicting-routes: "{{ .Values.global.blacklistConflictingRoutes }}"
```

---

Yes, it is fixed.